Src/WitsmlExplorer.Frontend/components/GlobalStyles.tsx

```diff
@@ -82,6 +97,41 @@ const GlobalStyles = createGlobalStyle`
       }
     }
   }
+
+  //over riding  input numbers padding from mterial ui
+  .MuiFormControlLabel-label {
```

Comment:
Regarding editing the global styles. New development should use the EDS component library rather than MUI, as we are in the process of replacing as much of MUI as possible. Also, I assume these styles are to match the sketches from the relevant GitHub issues. When using EDS, the default styling should take precedence over anything custom for a streamlined experience.